package.json

```diff
@@ -1,35 +1,39 @@
-{
+ {
   "name": "template-electron-vuex-vuetify",
+  "version": "0.1.0",
+  "private": true,
   "description": "Description",
-  "productName": "Product Name",
```

Comment:
we need a product Name for the apps as this appears in title bar and picked up by electron if we don't specify specific title in its config.

Reply:
never mind its at the bottom.